nss-day-cohort-48/fun-website-1-bugaboos
# Started on the index.html css

## Testing Instructions

1. git fetch --all
2. git checkout pk-css
3. serve

Make sure the home page looks nice
